pcsx2/Common.h

```diff
@@ -19,6 +19,8 @@
 
 static const u32 BIAS = 2;				// Bus is half of the actual ps2 speed
 static const u32 PS2CLK = 294912000;	//hz	/* 294.912 mhz */
+static s64 PSXCLK = 36864000;	/* 36.864 Mhz */
```

Comment:
@refractionpcsx2:
Should this be `extern s64 PSXCLK;` and, for example, in CDVD.cpp globally defined `s64 PSXCLK = 33868800;`?
I'm not sure.

Reply:
Hmm, not sure, I mean theoretically this is correct.  The alternative would be to have it in psxCommon.h and then have an extern reference in common.h.  I don't think it should be in CDVD.cpp though, since it is the global PSX clock, not a CDVD specific thing.

Reply:
Yeah, it must not be static, otherwise any value modifications will not be global.

Reply:
I guess it's a minor thing but when I see such a global, I never know where to initialize it. Ref is right, it doesn't belong into CDVD.cpp either. But where should it go? :p

Reply:
The issue is that I'm not getting an undefined external that clearly makes it obvious where to redefine it I'm getting a multiply defined symbol error

Reply:
Well it needs to be in either common.h or psxCommon.h, since it is the IOP clock speed, it only makes sense in those locations really, especially since the EE clock speed is in common.h :P

Reply:
Exactly the issue is where to redefine it see my previous comment I edited it

Reply:
Did you do as rama said? `extern s64 PSXCLK;` in Common.h and `s64 PSXCLK = 36864000;` in CDVD.cpp? (Let's just place it there for now even if it doesn't belong there, it can be moved later.)

The error sounds like you didn't add the `extern` in the header file.

Reply:
Done.